src/config/options.schema.json

```diff
@@ -2517,6 +2517,12 @@
               "description": "Check TD3 data structure invariants",
               "type": "boolean",
               "default": false
+            },
+            "parallel_domains" :{
+              "title": "solvers.td3.parallel_domains",
+              "description": "Maximal number of Domains that the solver can use in parallel. Only applies, when a solver of the 'td_parallel_*' family is used.",
+              "type": "integer",
+              "default": 2
```

Comment:
Seems like the existing `jobs` option would also work for this.

Reply:
I think it's worth having the concurrency settings for non-core stuff (g2html, preprocessing) separate from the solver concurrency, especially when comparing runtimes.